presto-hive/src/test/java/io/prestosql/plugin/hive/TestBackgroundHiveSplitLoader.java

```diff
@@ -496,7 +495,7 @@ public void testHive2VersionedFullAcidTableFails()
         for (String path : filePaths) {
             File file = new File(path);
             assertTrue(file.getParentFile().exists() || file.getParentFile().mkdirs(), "Failed creating directory " + file.getParentFile());
-            writeDeltaFileData(file);
+            assertTrue(writeFile(file), "Failed to write file");
```

Comment:
I liked your previous API better:
```
writeDeltaFileData(file);
```

make the method `void` again and add the assert inside, just for the `file.createNewFile()` case

Reply:
saw, you already fixed and merged, thank you.